delta/app/src/main/scala/ch/epfl/bluebrain/nexus/delta/routes/PluginsInfoRoutes.scala

```diff
@@ -0,0 +1,65 @@
+package ch.epfl.bluebrain.nexus.delta.routes
+
+import akka.http.scaladsl.server.Directives._
+import akka.http.scaladsl.server.Route
+import ch.epfl.bluebrain.nexus.delta.rdf.Vocabulary.contexts
+import ch.epfl.bluebrain.nexus.delta.rdf.jsonld.context.{ContextValue, RemoteContextResolution}
+import ch.epfl.bluebrain.nexus.delta.rdf.utils.JsonKeyOrdering
+import ch.epfl.bluebrain.nexus.delta.sdk.Permissions._
+import ch.epfl.bluebrain.nexus.delta.sdk.circe.CirceUnmarshalling
+import ch.epfl.bluebrain.nexus.delta.sdk.directives.AuthDirectives
+import ch.epfl.bluebrain.nexus.delta.sdk.directives.DeltaDirectives._
+import ch.epfl.bluebrain.nexus.delta.sdk.model.BaseUri
+import ch.epfl.bluebrain.nexus.delta.sdk.model.acls.AclAddress
+import ch.epfl.bluebrain.nexus.delta.sdk.model.search.Pagination.OnePage
+import ch.epfl.bluebrain.nexus.delta.sdk.model.search.SearchResults
+import ch.epfl.bluebrain.nexus.delta.sdk.model.search.SearchResults._
+import ch.epfl.bluebrain.nexus.delta.sdk.plugin.PluginInfo
+import ch.epfl.bluebrain.nexus.delta.sdk.{Acls, Identities}
+import io.circe.Encoder
+import io.circe.generic.semiauto.deriveEncoder
+import kamon.instrumentation.akka.http.TracingDirectives.operationName
+import monix.bio.UIO
+import monix.execution.Scheduler
+
+class PluginsInfoRoutes(identities: Identities, acls: Acls, info: List[PluginInfo])(implicit
```

Comment:
For now, it is ok but in the end it would be great to have this information merged with the other application information/health status

Reply:
yeah, maybe. Right now the `/version` endpoint shows version information about the services (cassandra, elasticsearch, blazegraph and remote storage service), which is not the same as the plugins.

But maybe we could have in them in the same endpoint, under a `plugins` json object. To be discussed

Reply:
I've added an issue for it https://github.com/BlueBrain/nexus/issues/1954